butane_core/src/codegen/migration.rs

```diff
@@ -69,9 +77,20 @@ fn many_table(main_table_name: &str, many_field: &Field, pk_field: &Field) -> AT
         .to_string();
     let many_field_type = get_many_sql_type(many_field)
         .unwrap_or_else(|| panic!("Mis-identified Many field {field_name}"));
+    let pk_field_name = pk_field
+        .ident
+        .clone()
```

Comment:
This can be `.as_ref` instead of `.clone()`. The `Option` needs to be consumed by expect, but its inner type does not.